packages/mermaid/src/utils.ts

```diff
@@ -941,6 +945,32 @@ export const insertTitle = (
     .attr('class', cssClass);
 };
 
+/**
+ * Parses a raw fontSize configuration value into a number and string value.
+ *
+ * @param fontSize - a string or number font size configuration value
+ *
+ * @returns parsed number and string style font size values, or nulls if a number value can't
+ * be parsed from an input string.
+ */
+export const parseFontSize = (fontSize: string | number | undefined): [number?, string?] => {
+  // if the font size is a number, assume a px string representation
+  if (typeof fontSize === 'number') {
+    return [fontSize, fontSize + 'px'];
+  }
+
+  const fontSizeNumber = parseInt(fontSize, 10);
+  if (Number.isNaN(fontSizeNumber)) {
+    // if a number value can't be parsed, return null for both values
+    return [null, null];
```

Comment:
Returning undefined here will keep the checks consistent with fontFamily & fontWeight
<img width="458" alt="image" src="https://user-images.githubusercontent.com/10703445/212625349-42b75cec-b646-4ed7-9794-1c06b7ad88d4.png">

Reply:
:wave: @sidharthv96 yep that's true, but returning undefined from a function that could be setting a "defined" response in all cases felt like a bad use of undefined.   Happy to change the new function to return undefined when a parsed value isn't available though, let me know!

Reply:
I agree with the reasoning. I also thought about it before commenting.
But, in cases like this where there isn't a clearly defined choice, I'd pick consistency across the codebase to make things easier. 

Not having to think of whether to use null or undefined and having to defend your choice might be why even the Official TS guide (for TS repo) included this rule.
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Reply:
👍 fair enough.  like I said, not a strong opinion 😄 .  updated in https://github.com/mermaid-js/mermaid/pull/3993/commits/9629c8d8d6a01dfda90fbc8d3eefe4eb8f1a8620